integr8ly/cloud-resource-operator
# MGDAPI-6417  CRO Release 1.1.4

## Overview

Jira: https://issues.redhat.com/browse/MGDAPI-6417

## Verification
1. Install and check standalone CRO 
- Clone this branch
- Run `make cluster/prepare`
- Run `make run`
- Check that CRO installed successfully

2. Check with Rhoam
- Create Index Image for CRO (from current PR)
- Create Index Image of RHOAM (see this Test PR: https://github.com/integr8ly/integreatly-operator/pull/3489)

3. What was done:
3.1. CRO standalone installation
3.2. RHOAM local install
3.3. RHOAM Upgrade TEST:  1.40.0 (CRO 1.1.3)  to 1.41.0 (CRO 1.1.4) on CCS cluster

```
$ oc get rhmi rhoam -n redhat-rhoam-operator -o json | jq -r '.status.stage'
complete

$ oc get rhmi rhoam -n redhat-rhoam-operator -o json
....

                    "cloud-resources": {
                        "host": "",
                        "name": "cloud-resources",
                        "operator": "1.1.4",
                        "status": "completed",
                        "version": "1.1.4"
                    },
.....
 "version": "1.41.0"
```
